src/cleanup.ts

```diff
@@ -105,4 +116,33 @@ function getWorkingDirectory(): string {
     return workingDirectory;
 }
 
+async function checkConnectionToArtifactory(): Promise<boolean> {
+    try {
+        core.startGroup('Checking connection to JFrog Artifactory');
+        const pingResult: string = await Utils.runCliAndGetOutput(['rt', 'ping']);
+        if (pingResult !== 'OK') {
+            core.warning('Could not connect to Artifactory. Skipping Build Info post tasks.');
```

Comment:
The pingResults are ignored in this scenario. Let's print it in log level debug:
```suggestion
            core.debug(`Ping result: ${pingResult}`);
            core.warning('Could not connect to Artifactory. Skipping Build Info post tasks.');
```